spec_helper.rb

```diff
@@ -48,10 +48,7 @@
   end
 end
 
-dir = "../fixtures/code"
-CODE_LOADING_DIR = defined?(require_relative) ?
-                     File.realpath(dir, __FILE__) :
-                     File.expand_path(dir, __FILE__)
+CODE_LOADING_DIR = File.expand_path "../fixtures/code", __FILE__
```

Comment:
@eregon ooh, I see now. @elia could use your help there :sweat_smile: Basically, as I mentioned above, realpath works, but returns a shorter path than expected, making the require fail.